test/SignatureValidator/Setup.t.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import { Test } from "forge-std/Test.sol";
+import { SignatureValidator } from "contracts/utils/SignatureValidator.sol";
+import { IERC1271Wallet } from "contracts/interfaces/IERC1271Wallet.sol";
+import { ECDSA } from "@openzeppelin/contracts/cryptography/ECDSA.sol";
+
+contract TestSignatureValidator is SignatureValidator, Test {
```

Comment:
Maybe not inherit from `SignatureValidator` and deploy it as a standalone contract so the tests won't have unexpected behavior?

Reply:
Fixed 21e54ebc9e203b6b082fff497bbbf49fbb9a6e05